src/main/webapp/index.html

```diff
@@ -66,32 +74,35 @@
             <small id="slider-output" name="slider-output">100</small>
         </div> 
       </div>
-      <div id="video-results" style="margin-top:20px">
-        <div class="row">
-          <div class="col-8">
-            <div id= "tab_list" class="list-group" role="tablist" ></div>
-          </div>
-          <div class="col-4">
-            <div id= "info_list" class="tab-content" id="nav-tabContent"></div>
+      <div id="search-analysis">
+        <div id="video-context"></div>  
+        <div id="chart-container"></div>
+        <div id="average-score-container"></div> 
+        <div id="word-cloud-container"></div> 
+        <div class="d-flex justify-content-center" id="video-results-loading-container">
+          <div id="loading-img" class="spinner-border text-dark" role="status" style="display:none">
+            <span class="sr-only">Loading...</span>
           </div>
         </div>
-      </div>
-      <div id="video-context"></div>
-      <div class="d-flex justify-content-center" id="video-results-loading-container">
-        <div id="loading-img" class="spinner-border text-dark" role="status" style="display:none">
-          <span class="sr-only">Loading...</span>
+        <div id="error-surfacer" style="display:none">
+          <h4 id="error-display-text"> Something went wrong :(<br> Try another link? </h4>
+          <h5 id="error-details"><h5>
         </div>
       </div>
-      <div id="error-surfacer" style="display:none">
-        <h4 id="error-display-text"> Something went wrong :(<br> Try another link? </h4>
-        <h5 id="error-details"><h5>
-      </div>
-      <div id="analysis-container">
-        <div id="sentiment-info">  
-          <div id="chart-container"></div>
-          <div id="average-score-container"></div>
-        </div>
+      <div id="link-analysis">
+        <div id="video-context"></div>  
```

Comment:
HTML element ids must be unique per page. (https://www.w3schools.com/html/html_id.asp)

While modern browsers support displaying elements w/ duplicate ids, having duplicate ids makes programmatic access of elements more difficult (i.e. why you have to use JQuery to find the correct elements - since `findElementById` now has undefined behaviour). 

Instead what you can do to make this logic simpler is to define seperate divs for the things that are shared between the two different tabs.
I.e.
`keyword-video-context` and `link-video-context`
`keyword-chart-container` and `link-search-container`.

To make this approach work with the current code, I think the easiest thing you can do is modify the functions you've changed to supply a element prefix instead of a div-id (so perhaps either "keyword" or "link") and replace the usages of jquery with string concatenation to determine the correct element id.

I.e.
`const chartContainer = $('#' + divId + '> #chart-container');`

could become:
`const ChartContainer = getElementById(prefix + '-chart-container')`

Or if you wanted to wrap this string concatenation in another (new) method could be:
`const ChartContainer = getElementByIdWithPrefix(prefix, 'chart-container')`

Does that make sense?